### DISCUSSION TITLE
Is there a way to extract special characters like ® from a pdf and write them in a txt?

### DISCUSSION THREAD

**Discussion by forna7** [pymupdf/PyMuPDF#2056] [Looking for help]
Hello, i am currently working on a text extraction program to make a bigger infrastucture,
is there a way to not lose these type of special characters in the process? I am pretty new to pymupdf so please forgive my ignorance

**Comment by JorjMcKie** [comment]
These characters are not "special" in any way and you should not loose them. All characters are treated the same - how could it be otherwise.
Text extraction will extract whatever can be extracted and treats every character (i.e. unicode) democratically 😉.
